cored/src/commonMain/kotlin/com/github/kittinunf/cored/Reducer.kt

```diff
@@ -0,0 +1,44 @@
+package com.github.kittinunf.cored
+
+import kotlin.reflect.KClass
+
+typealias AnyReducer<S> = Reducer<S, Any>
+typealias ReducerType<S, A> = Pair<KClass<out Any>, Reducer<S, A>>
+
+enum class Order {
+    BeforeReduce,
+    AfterReduced
```

Comment:
`AfterReduced` or `AfterReduce`? 🤔

Reply:
Yeah ... it think it should have been `AfterReduce`*